### ISSUE TITLE
What about Resource Class?

### ISSUE THREAD

**Issue by MeirPeleg** [MicrosoftDocs/sccm-docs-powershell-ref#135]
You haven't mention anything about it

When creating the query, it doesn't specify "Object Type" and the query not ruining, how can we sett the "System Resource"?
---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: b280b2a7-09a5-8fbe-563d-f5a04bce7f29
* Version Independent ID: 99c826a2-14ba-c2dc-b207-172370c2c188
* Content: [New-CMQuery - Configuration Manager](https://docs.microsoft.com/en-us/powershell/module/configurationmanager/new-cmquery?view=sccm-ps#feedback)
* Content Source: [sccm-ps/ConfigurationManager/New-CMQuery.md](https://github.com/MicrosoftDocs/sccm-docs-powershell-ref/blob/master/sccm-ps/ConfigurationManager/New-CMQuery.md)
* Product: **configuration-manager**
* Technology: **configmgr-other**
* GitHub Login: @aczechowski
* Microsoft Alias: **aaroncz**

**Comment by MeirPeleg** [comment]
Run this for example:

`new-CMQuery -Name "Server 2016" -Expression  'select SMS_R_System.Name, SMS_R_System.LastLogonUserName, SMS_G_System_OPERATING_SYSTEM.Caption from  SMS_R_System inner join SMS_G_System_OPERATING_SYSTEM on SMS_G_System_OPERATING_SYSTEM.ResourceID = SMS_R_System.ResourceId where SMS_G_System_OPERATING_SYSTEM.Caption like "Microsoft Windows Server 2012%"' -TargetClassName 'System Resource' -LimitToCollectionId:$false

**Comment by aczechowski** [comment]
Hi @MeirPeleg, thanks for the feedback. The **TargetClassName** parameter accepts the actual class _name_, not the description. So for "System Resource" it's the **SMS_S_System** class name. If you use `-TargetClassName "SMS_R_System"` it correctly sets the object type.

Also, I figured out the format for the **LimitToCollectionId** parameter:
- To use a limiting collection, specify the collection's ID
- To be not collection limited, set this parameter to a blank string `""` (I tried `$null` but that didn't work, only an empty string)
- To prompt for a collection, set this parameter to `"<Prompt>"`

I'm updating the article (and all related articles for CMQuery) with this information.
